### PR TITLE
s/Travis CI/GitHub Actions

### PR DESCRIPTION
**Pull request by trentm**
Refs: https://github.com/elastic/apm-agent-nodejs/issues/1922

### CODE REVIEW

**Comment by trentm** [comment]
Hrm. I don't know why my added GitHub action is not being run here. Interplay with having other checks already perhaps?

**Comment by trentm** [comment]
GH action is running now. Hrm. Perhaps just GH Actions missing some events? Dunno.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  pull_request:
+    branches:
+    - master
+jobs:
+  test-vers:
+    strategy:
+      matrix:
+        # Test with the latest LTS.
+        node: ['14']
+    runs-on: ubuntu-latest
```
Comment (astorm) [inline .github/workflows/test.yml:16]: I'm leaning things about travis right up to the end -- `lts/*` is "the latest node LTS version": https://docs.travis-ci.com/user/languages/javascript-with-nodejs/ 👍

**Review by astorm — APPROVED** [review]
Does the thing, approving.

**Pull request merged**